src/components/SingleCoupon/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Typography, Button, Row, Col, Card } from "antd";
+
+const { Title } = Typography;
+
+const SingleCoupon = ({
+  coupon,
+  deleteSingleCoupon,
+  setTotalAmount,
+  changeAmount,
+  isTesting,
+}) => {
+  const [amount, setAmount] = useState(1);
+
+  return (
+    <div>
+      <Row>
+        <Col>
+          <Card hoverable align="center" className="couponCards">
+            {isTesting ? (
+              <button type="button">x</button>
+            ) : (
+              <Button
```

Comment:
Can you put this button on the right, not on the middle ?

Reply:
in right, it's braking style, now it's on the left